src/wi_stuff.c

```diff
@@ -559,7 +559,8 @@ static boolean DrawAnimation(void)
     if (animation->background_lump)
     {
         V_DrawPatchFullScreen(
-            V_CachePatchName(animation->background_lump, PU_CACHE));
+            V_CachePatchName(
+              W_CheckWidescreenPatch(animation->background_lump), PU_CACHE));
```

Comment:
This only works for ID24 interlevel definition. We also do not need it because all ID24 ports support widescreen graphics.